_candidates_en/kmizu_1.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Kota Mizushima (kmizu)
+title: "Gentle introduction to DOT calculus"
+length: 40
+audience: Advanced
+language: Japanese
+twitter: kmizu
+github: kmizu
+icon: https://pbs.twimg.com/profile_images/703094910417678337/b_v8jTJ3_400x400.jpg
+organization: Dwango, Inc
+tags:
+  - Functional Programming, Others
+suggestions:
+  - Those interested in programming language theory
+  - Those interested in the theoretical foundation of Scala
+  - Those who want to understand Scala deeoper
+contributes:
+  - fialyzer (https://github.com/dwango/fialyzer)
+  - Onion (https://github.com/onion-lang/onion)
+  - Klassic (https://github.com/klassic/klassic)
+speaker_experience:
+  - Scala Days 2010
+  - ScalaMatsuri 2018
+  - Scala Fukuoka 2019
+---
+The Dependent Object Types (DOT) calculus is the theory on which Dotty (which will be Scala 3.0) is based.  The DOT calculus is a model to understand the theoretical backbone of Scala. Such a model is called a "core calculus", and it is used to prove the characteristics of a programming language. In this session, I'll give a gentle introduction to the DOT calculus.
```

Comment:
Line 26 Original: - The Dependent Object Types (DOT) calculus is the theory on which Dotty (which will be Scala 3.0) is based.  The DOT calculus is a model to understand the theoretical backbone of Scala. Such a model is called a "core calculus", and it is used to prove the characteristics of a programming language. In this session, I'll give a gentle introduction to the DOT calculus.
Change: - The Dependent Object Types (DOT) calculus is the theory on which Dotty (which will be Scala 3.0) is based. The DOT calculus is a model to understand the theoretical backbone of Scala. Such a model is called a "core calculus", and it is used to prove the characteristics of a programming language. In this session, I'll give a gentle introduction to the DOT calculus.